gum/guminterceptor.c

```diff
@@ -1782,7 +1897,7 @@ gum_function_context_fixup_cpu_context (GumFunctionContext * function_ctx,
 }
 
 static InterceptorThreadContext *
-get_interceptor_thread_context (void)
```

Comment:
This is incorrect; in C (unlike C++), `(void)` means "no arguments", whereas `()` means "0 or more arguments".